kong/pdk/ip.lua

```diff
@@ -1,5 +1,25 @@
+---
+-- Trusted IPs module
+--
+-- This module allows querying whether an IP is trusted or not for X-Forwarded-* headers.
+-- It depends on the `trusted_ips` option in the Kong config file
```

Comment:
"the `trusted_ips` configuration property" is how we generally refer to... configuration properties (at least on the website). Since they do not necessarily come from the config file (env variables and CLI arguments too). Ditto in the below paragraph.